common/src/main/scala/services/teamwork/SquadService.scala

```diff
@@ -0,0 +1,3017 @@
+// Copyright (c) 2019 PSForever
```

Comment:
I wont nitpick this file, plus it's hard to review given the size. Nice job!

Reply:
I do plan to break it up.  This is just the only way I could deal with the initial setup.

Reply:
That's okay. when we come up with a plan to breakup WSA, that can propagate to the monolithic services.